packages/cli/src/commands/config/set.ts

```diff
@@ -44,13 +46,15 @@ export default class Set extends Command {
     }),
   };
 
+  @AuthenticationRequired()
```

Comment:
To be able to list the organization the users has access to, you need to be logged in.

Reply:
Tho will need to be updated when #122 will be merged